Week3/app/src/main/res/layout-land/activity_main.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Nice work constructing a landscape view and incorporating a scrollview so that the entire layout is visible.